src/FakeItEasy/Configuration/IAfterCallSpecifiedConfiguration.cs

```diff
@@ -3,8 +3,9 @@ namespace FakeItEasy.Configuration
     /// <summary>
     /// Lets you set up expectations and configure repeat for the configured call.
     /// </summary>
-    public interface IAfterCallSpecifiedConfiguration
-        : IRepeatConfiguration

```

Comment:
I'm good with keeping. Just wondering if we'd made it skinnier over the years or something. I should've looked in the gits.

Good point about the name! We could add that to our future "breaking changes" list.